components/brave_wallet_ui/page/screens/fund-wallet/fund-wallet.tsx

```diff
@@ -252,32 +278,33 @@ export const FundWalletScreen = () => {
 
                 {assetsForFilteredNetwork.length
                   ? <TokenLists
-                      enableScroll
-                      maxListHeight='38vh'
-                      userAssetList={assetsForFilteredNetwork}
-                      networks={networksFilterOptions}
-                      hideAddButton
-                      hideAssetFilter
-                      hideAccountFilter
-                      estimatedItemSize={100}
-                      renderToken={({ item: { asset } }) => {
-                        return <BuyAssetOptionItem
-                          isSelected={asset === selectedAsset}
-                          key={asset.isErc721
-                            ? `${asset.contractAddress}-${asset.symbol}-${asset.chainId}`
-                            : `${asset.contractAddress}-${asset.tokenId}-${asset.chainId}`}
-                          token={asset}
-                          tokenNetwork={getTokensNetwork(buyAssetNetworks, asset)}
-                          onClick={setSelectedAsset}
-                        />
-                      }}
-                    />
+                    enableScroll
+                    maxListHeight='38vh'
+                    userAssetList={assetsForFilteredNetwork}
+                    networks={networksFilterOptions}
+                    hideAddButton
+                    hideAssetFilter
+                    hideAccountFilter
+                    estimatedItemSize={100}
+                    renderToken={({
+                      index,
+                      item: { asset }
+                    }) => <BuyAssetOptionItem
+                        isSelected={checkIsBuyAssetSelected(asset)}
+                        key={asset.isErc721
+                          ? `${asset.contractAddress}-${asset.symbol}-${asset.chainId}`
+                          : `${asset.contractAddress}-${asset.tokenId}-${asset.chainId}`}
+                        token={asset}
+                        tokenNetwork={getTokensNetwork(buyAssetNetworks, asset)}
+                        onClick={setSelectedAsset}
+                        ref={(ref) => refs.current[index] = (checkIsBuyAssetSelected(asset) ? ref : null)} />}
```

Comment:
Is an array of refs needed if we are only targeting one item?

Reply:
Cleaned this up a bit, now only references on ref to scroll into view!